test/panic/util/open.js

```diff
@@ -0,0 +1,34 @@
+module.exports = {
+    web: (count, url, options) => {
+        if (typeof count === 'string') {
+            options = url;
+            url = count;
+            count = 1;
+        }
+        if (!url || typeof url !== 'string') {
+            throw new Error('Invalid URL');
+        }
+        try {
+			require('puppeteer').launch(options).then(browser => {
+				Array(count).fill(0).forEach((x, i) => {
+					console.log('Opening browser page ' + i + ' with puppeteer...');
+					browser.newPage().then(page => {
+						page.on('console', msg => {
+							if (msg.text() === 'JSHandle@object') {
+								// FIXME: Avoid text comparison
+								return;
+							}
+							console.log(`${i} [${msg.type()}]: ${msg.text()}`);
+						});
+						return page.goto(url);
+					}).then(() => {
+						console.log('Browser page ' + i + ' open');
+					});
+				});
+			});
+		} catch (err) {
+			console.log("PLEASE OPEN "+ url +" IN "+ count +" BROWSER(S)!");
+			console.warn('Consider installing puppeteer to automate browser management (npm i -g puppeteer && npm link puppeteer)');
```

Comment:
When only having api usage (not cli usage), you could install it as a dev dependency. This would remove the need for the host to install puppeteer globally.

Reply:
I considered adding it to dev dependencies, but at this stage, panic tests are not run automatically and I'm guessing most people don't run them at all. Puppeteer downloads Chromium as a post install script which is about 125 MB, so it would burden a standard dev setup.

The global/link combo has the added benefit of easily switching it on and off via link/unlink even when working offline.